### PR TITLE
Ensure that single query used for event lookup

### LINKED ISSUE #1: Make get_events more efficient for batches

### PR DESCRIPTION
**Pull request by stephenbinns**
Resolves #1

### CODE REVIEW

File: lib/entity_store_sequel/postgres_entity_store.rb [hunk 1]
```diff
@@ -195,18 +195,23 @@ def get_entities(ids, options={})
     def get_events(criteria)
       return {} if criteria.empty?
 
-      result = {}
+      query = events.where('1=2')
```
Comment (stephenbinns) [inline lib/entity_store_sequel/postgres_entity_store.rb]: Ugly hack but Sequel's query building is rather clunky when greater than comparisons are needed

Comment (gshutler) [reply lib/entity_store_sequel/postgres_entity_store.rb]: May be able to build the query without a hack by using a [union](http://www.rubydoc.info/github/jeremyevans/sequel/Sequel%2FDataset%3Aunion). It is weird there isn't something like a `none` to give you an empty set to add to like this essentially does.

Comment (gshutler) [reply lib/entity_store_sequel/postgres_entity_store.rb]: That made me think of a different query to Google and I finally found something: http://sequel.jeremyevans.net/rdoc-plugins/files/lib/sequel/extensions/null_dataset_rb.html

Comment (stephenbinns) [reply lib/entity_store_sequel/postgres_entity_store.rb]: I didn't have much success with nullify - I've gone with a ruby solution and just dynamically invoked the methods.

**Comment by gshutler** [comment]
👍

**Pull request merged**